finetuner/run.py

```diff
@@ -99,6 +99,45 @@ def stream_logs(self, interval: int = 5) -> Iterator[str]:
             experiment_name=self._experiment_name, run_name=self._name
         )
 
+    def metrics(self) -> Dict:
```

Comment:
return type hint? `Dict[str, float]`?

Reply:
no its `Dicct[str, Dict[str, float]]` but yes I can add it